modules/server/src/main/ts/bedrock/auto/Driver.ts

```diff
@@ -67,6 +68,14 @@ const addArguments = (capabilities: Record<string, any>, name: string, args: str
   capabilities[name].args = currentArgs.concat(args);
 };
 
+const getExtraBrowserCapabilities = (settings: DriverSettings): string[] => {
+  if (settings.extraBrowserCapabilities && settings.extraBrowserCapabilities.length > 0) {
```

Comment:
Will this ever be undefined? Doesn't the default value of the ClOptions set this as an empty string? If it can be undefined then it should be marked with a ? in the types.

Reply:
I'll test if it is defaulting it properly. I tried a few things to get around the -- issue, and this was probably an artifact of that.

- [ ] see if I can remove the falsy check.

Reply:
Removed the falsy check.